query/raw_query.go

```diff
@@ -39,6 +39,15 @@ func newParsedRawQuery(q *ParsedQuery) (ParsedRawQuery, error) {
 	return rq, nil
 }
 
+// TimestampFieldIndex returns the index of the timestamp field.
+func (q *ParsedRawQuery) TimestampFieldIndex() int { return 0 }
```

Comment:
nit: Should we comment these magic numbers?

Reply:
This is actually documented in the `computeFieldConstraints` function for the respective queries.